typescript/packages/plugins/steer-protocol/src/parameters.ts

```diff
@@ -0,0 +1,94 @@
+import { z } from "zod";
+
+/**
+ * Parameters schema for depositing tokens into a Steer Smart Pool
+ */
+export const depositParametersSchema = z.object({
+    // Pool address where liquidity will be deposited
+    poolAddress: z.string().describe("The address of the Steer Smart Pool"),
+
+    // Amount of token0 to deposit
+    amount0Desired: z.string().describe("The desired amount of token0 to deposit"),
+
+    // Amount of token1 to deposit
+    amount1Desired: z.string().describe("The desired amount of token1 to deposit"),
+
+    // Minimum amount of token0 to accept (slippage protection)
+    amount0Min: z
+        .string()
+        .describe(
+            "The minimum amount of token0 to deposit, in base units (e.g. wei, not ether). Used for slippage protection.",
+        )
+        .default("0"),
+
+    // Minimum amount of token1 to accept (slippage protection)
+    amount1Min: z
+        .string()
+        .describe(
+            "The minimum amount of token1 to deposit, in base units (e.g. wei, not ether). Used for slippage protection.",
+        )
+        .default("0"),
+
+    // Optional deadline for the transaction
+    deadline: z
+        .number()
+        .optional()
+        .default(60 * 60 * 24)
+        .describe("Optional deadline for the transaction"),
+});
+
+/**
+ * Parameters schema for withdrawing tokens from a Steer Smart Pool
+ */
+export const withdrawParametersSchema = z.object({
+    // Pool address to withdraw from
+    poolAddress: z.string().describe("The address of the Steer Smart Pool"),
+
+    // Number of LP shares to withdraw
+    shares: z.string().describe("The number of pool shares to withdraw"),
+
+    // Minimum amount of token0 to receive (slippage protection)
+    amount0Min: z
+        .string()
+        .describe(
+            "The minimum amount of token0 to deposit, in base units (e.g. wei, not ether). Used for slippage protection.",
+        )
+        .default("0"),
+
+    // Minimum amount of token1 to receive (slippage protection)
+    amount1Min: z
+        .string()
+        .describe(
+            "The minimum amount of token1 to deposit, in base units (e.g. wei, not ether). Used for slippage protection.",
+        )
+        .default("0"),
+
+    // Optional deadline for the transaction
+    deadline: z
```

Comment:
Is deadline even used? I see its not used when sending the tx

Reply:
I will remove this.